cli/src/main.rs

```diff
@@ -102,6 +103,14 @@ enum CLI {
         /// The path to the file to be downloaded to the flash
         path: String,
     },
+    /// Download memory to attached target
+    #[structopt(name = "d")]
+    D {
```

Comment:
I think we could just remove the other download method, this is a bit confusing right now. And d is not really a telling name.